ngraph/core/include/ngraph/op/assign.hpp

```diff
@@ -23,10 +23,33 @@ namespace ngraph
 {
     namespace op
     {
+        class NGRAPH_API AssignBase : public Sink
+        {
+        public:
+            NGRAPH_RTTI_DECLARATION;
+            AssignBase() = default;
+            /// \brief Constructs an AssignBase operation.
+            explicit AssignBase(const OutputVector& arguments)
+                : Sink(arguments)
+            {
+            }
+            virtual std::shared_ptr<ngraph::Variable> get_variable() const { return m_variable; }
```

Comment:
Please add doxygen comment to all public methods.

Reply:
fixed